example/C/src/Rhythm/SensorSimulator.lf

```diff
@@ -5,24 +5,27 @@
 target C {
     threads: 2,
     flags: "-lncurses",
+    cmake: false,
     files: ["/lib/C/util/sensor_simulator.c", "/lib/C/util/sensor_simulator.h"]
```

Comment:
How are entries of the `files` directive resolved? Where is it determined that `/` is supposed to mean `org.lflang/src/`?

Reply:
This is not part of this PR.  This is how it has been for a while now.

Reply:
I'm aware of that, which is why I made this comment and approved the PR nonetheless. That said, this code looks suspicious, and it is not obvious to me how this is supposed to work in general.